app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,58 @@
     font-weight: normal;
   }
 }
+
+ul.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+ul.list-of-rats li{
```

Comment:
Merge rule `ul.list-of-rats li` with rule on line 40<br>Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space